### ISSUE TITLE
Collection of "Track Recoder" issues

### ISSUE THREAD

**Issue by Rob-from-VI** [organicmaps/organicmaps#9554]
This seems to have fallen off the radar a bit.   I just wanted to summarize what I feel are the current main outstanding isses to improve and perfect this feature, and see if others agree as to priority and completenes (wishful thinking) of this list as follows:

**Following are most urgent**

Feature to adjust precision ie point spacing/timing, (which can also be used to decrease storage space, and to reduce battery use) see also #9342

Ensure that the exact starting point, and the exact final point when saving (both with times) are recorded.  **Update 2024 11 18 this is still not working right, I will post a new issue for it.  see #9685**

- [x] Feature to allow saving to a selected list or folder, and to edit name of the track when saving  See also #9680  https://github.com/organicmaps/organicmaps/pull/9033

- [ ] Ability to display or hide on map individual tracks within a list https://github.com/organicmaps/organicmaps/issues/5503

- [ ] Ability to export individual recorded tracks within a list https://github.com/organicmaps/organicmaps/issues/9283 , see also #9580

Ability to save exported track to SD card when present.  (Currently I am only able to save to internal memory)  **Update - as of 2024 10 release, this seems to be working now.**

**Less urgent**

Display of information about a selected track in a list, including Distance, time taken, average speed, elevation gain, etc  -  see also #9573

Ability to "select" a displayed track onscreen, and display the above information

Display of the above information start to present for a track while being recorded  -  See also #9678 ,  #9524

**Possible future edit features**

Ability to edit tracks on screen such as remove extraneous tail when we forget to stop and save.

Ability to relocate stray recorded points along a track, add or delete points

**Comment by pastk** [comment]
Some of the points have separate issues already. It'd be helpful to link them (I did a few as an example).

https://github.com/organicmaps/organicmaps/issues?page=1&q=is%3Aopen+is%3Aissue+label%3A%22Track+Recording%22

https://github.com/organicmaps/organicmaps/issues?q=is%3Aopen+is%3Aissue+label%3A%22Bookmarks+and+Tracks%22+track

**Comment by pastk** [comment]
> Ensure that the exact starting point, and the exact final point when saving (both with times) are recorded.

If they're not then its a bug. Did you have such cases?

**Comment by Aurareus** [comment]
Existing issues for each item listed here:

> Feature to adjust precision ie point spacing/timing, (which can also be used to decrease storage space, and to reduce battery use)

#9342

> Ensure that the exact starting point, and the exact final point when saving (both with times) are recorded.

Related: #8703 

> Feature to allow saving to a selected list or folder, and to edit name of the track when saving

#9284 #9275 #5512 #5680 Related: #9411 

> Ability to display or hide on map individual tracks wihtthin a list

#5503 Related: #5511 

> Ability to export individual recorded tracks within a list

#9283 #9275

> Display of information about a selected track in a list, including Distance, time taken, average speed, elevation gain, etc
> Ability to "select" a displayed track onscreen, and display the above information
> Display of the above information start to present for a track while being recorded

#9275 #2353 Related: #5388 #6528 #8605 #5579

**Comment by Rob-from-VI** [comment]
> > Ensure that the exact starting point, and the exact final point when saving (both with times) are recorded.
> 
> If they're not then its a bug. Did you have such cases?

Yes, absolutely.   I tested this several times using last beta, and rarely or never is the starting point nor the ending point precisely recorded.   If my last recorded point is say 10m from my final destination, I can pace back and forth around my stopping point for a long time, and no point will be set there, unless I move at least 16m from my last recorded point.

As to starting point, I start recording zoomed in on a known point, and the first recorded point, ie the beginning of the dotted line will always start some distance from where I pressed "start record."

**Comment by Aurareus** [comment]
I can also confirm I am experiencing the same behavior.
I have been testing the track recorder in tandem with the track recording on other apps to record in parallel and have noticed that the recorded tracks from Organic Maps sometimes (but not always) have a later start point than the tracks recorded in the other app.

**Comment by Aurareus** [comment]
> Display of information about a selected track in a list, including Distance, time taken, average speed, elevation gain, etc
Ability to "select" a displayed track onscreen, and display the above information
Display of the above information start to present for a track while being recorded

As there was no dedicated issue specifically for this I have opened a feature request #9573
I've also added it to the above list.

There is also currently no dedicated issue open (or at least none that I can find) for these:

> Ensure that the exact starting point, and the exact final point when saving (both with times) are recorded.

> Ability to save exported track to SD card when present. (Currently I am only able to save to internal memory)

> Ability to edit tracks on screen such as remove extraneous tail when we forget to stop and save.

> Ability to relocate stray recorded points along a track, add or delete points

**Comment by pastk** [comment]
> Yes, absolutely. I tested this several times using last beta, and rarely or never is the starting point nor the ending point precisely recorded. If my last recorded point is say 10m from my final destination, I can pace back and forth around my stopping point for a long time, and no point will be set there, unless I move at least 16m from my last recorded point.
> 
> As to starting point, I start recording zoomed in on a known point, and the first recorded point, ie the beginning of the dotted line will always start some distance from where I pressed "start record."

Thanks! Could you please open a separate bug issue for it?

**Comment by rtsisyk** [comment]
Thanks for collecting track recorder issues. Please feel free to use https://github.com/organicmaps/organicmaps/issues?q=state%3Aopen%20label%3A%22Track%20Recording%22 to mark all such issues. There is no need for a meta ticket.
